pkg/schedulingtypes/plugin.go

```diff
@@ -129,13 +123,28 @@ func (p *Plugin) HasSynced() bool {
 		glog.V(2).Infof("Cluster list not synced")
 		return false
 	}
+
+	if !p.templateController.HasSynced() {
+		return false
+	}
+	if !p.placementController.HasSynced() {
+		return false
+	}
+	if !p.overrideController.HasSynced() {
+		return false
+	}
+
 	clusters, err := p.targetInformer.GetReadyClusters()
 	if err != nil {
 		runtime.HandleError(fmt.Errorf("Failed to get ready clusters: %v", err))
 		return false
 	}
 
-	return p.targetInformer.GetTargetStore().ClustersSynced(clusters)
+	if !p.targetInformer.GetTargetStore().ClustersSynced(clusters) {
```

Comment:
Is this change for readability or am missing something?

Reply:
Yes. When I was adding the checks for the template/placement/override controllers having synced, I thought it made sense for every failed check to return false and then a default true be returned so that no matter the ordering or changes to the checks the true would always be the default.  I'm happy to revert if you'd like.

Reply:
Makes more sense this way. Thanks for explaining.